task_5.1.py

```diff
@@ -0,0 +1,23 @@
+# Multi-paradigm programming languages, task №5.1
+# Andrey Fortus IKM-221a
+
+GENERAL_INFO = 'Multi-paradigm programming languages, task №5.1'
+STUDENT_INFO = 'Andrey Fortus IKM-221a variant №20'
+
+# info
+print(GENERAL_INFO)
+print(STUDENT_INFO)
+
+
+# function for finding the sum of a series
+def sum_series(n):
+    suma = 1  # start value
+    while 1 / pow(2, n) + 1 / pow(3, n) >= 1e-4:  # 1e-4 = 1 * 10−4
+        suma += 1 / pow(2, n) + 1 / pow(3, n)
```

Comment:
move the calculation to a separate variable

Reply:
consider one more solution:


from itertools import takewhile, count, repeat

row = (1 / pow(2, n) + 1 / pow(3, n) for n in count())
print(sum(list(takewhile(lambda x: x >= 1e-4, row))))